tests/integration-tests/tests/runtime_bake/test_runtime_bake/test_runtime_bake/pre-install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -x
+
+. /etc/os-release
+release="${ID}${VERSION_ID:+.${VERSION_ID}}"
+
+if [ `echo ${release} | grep '^centos\.7'` ]; then
+  os='centos7'
+elif [ `echo "${release}" | grep '^centos\.8'` ]; then
+  os='centos8'
+else
+  os='other'
```

Comment:
I think it would be good to fail here, because it would be much easier to debug an explicit failure here and change the OS detection logic instead of having to trace a failure during the runtime baking process back to a silent failure. This would require not running this script for OSs other than centos7 and centos8.